DEVELOPING.rst

```diff
@@ -8,461 +8,91 @@
-  branch;
-
-- for new developments, you would typically start your topical branch
-  from the ``master`` branch.
-
-Here is example:
-
-.. code-block:: console
-
-   $ git checkout master
-   $ git checkout -b improve-event-display-icons
-   $ $EDITOR some_file.py
-   $ git commit -a -m 'some improvement'
-   $ $EDITOR some_other_file.py
-   $ git commit -a -m 'some other improvement'
-
-When everything is ready, you may want to rebase your topical branch
-to get rid of unnecessary commits:
+4. If you want to load only some records/docs
 
 .. code-block:: console
 
-   $ git checkout improve-event-display-icons
-   $ git rebase master -i # squash commits here
-
-Making pull requests
---------------------
-
-You are now ready to issue a pull request: just push your branch in
-your personal repository:
-
-.. code-block:: console
-
-   $ git push origin improve-event-display-icons
-
-and use GitHub's "Pull request" button to make the pull request.
-
-Watch GitHub Actions build status report to see whether your pull request
-is OK or whether there are some troubles.
-
-Updating pull requests
-----------------------
-
-Consider the integrator had some remarks about your branch and you
-have to update your pull request.
-
-Firstly, update to latest upstream "master" branch, in case it may
-have changed in the meantime:
-
-.. code-block:: console
-
-   $ git checkout master
-   $ git fetch upstream
-   $ git merge upstream/master --ff-only
-
-Secondly, make any required changes on your topical branch:
-
-.. code-block:: console
-
-   $ git checkout improve-event-display-icons
-   $ $EDITOR some_file.py
-   $ git commit -a -m 'amends something'
-
-Thirdly, when done, interactively rebase your topical branch into
-nicely organised commits:
-
-.. code-block:: console
-
-   $ git rebase master -i # squash commits here
-
-Finally, re-push your topical branch with a force option in order to
-update your pull request:
-
-.. code-block:: console
-
-   $ git push origin improve-event-display-icons -f
-
-Finishing pull requests
------------------------
-
-If your pull request has been merged upstream, you should update your
-local sources:
-
-.. code-block:: console
-
-   $ git checkout master
-   $ git fetch upstream
-   $ git merge upstream/master --ff-only
-
-You can now delete your topical branch locally:
-
-.. code-block:: console
+   $ my_docker exec -it web cernopendata fixtures records --file /content/data/records/<full_path>
+   $ my_docker exec -it web cernopendata fixtures docs --file /content/data/docs/<full_path>
 
-   $ git branch -d improve-event-display-icons
+..
 
-and remove it from your repository as well:
+5. Finally, if there are new images, ensure that they appear in the correct folder
 
 .. code-block:: console
 
-   $ git push origin master
-   $ git push origin :improve-event-display-icons
+   $ my_docker exec -it web /data/scripts/load-images.sh
 
-This would conclude your work on ``improve-event-display-icons``.
+..
```

Comment:
It would be more natural if the description of "how to add a new field" is in the DEVELOPING guide of the repository that offers the schema, isn't it? From this one, we can add a link to that developing guide

Reply:
Sure, such a link would perfectly do. My main aim was to not leave this subject topic "empty" here.

Reply:
Isn't the link in line 59 enough for that? That mentions as well the schema.

Reply:
That link seems to point just to the `cernopendata-portal` repository home, which does not mention much about how to add new field or a new mapping.  (Recall the old user story "how do I add a new magnet polarity field".) Sure, we can write these stories later, but it should not be forgotten... I thought the split is a perfect occasion to do that.

(BTW https://github.com/cernopendata/cernopendata-portal/blob/main/DEVELOPING.rst needs updating too.)

Reply:
The updates to the cernopendata-portal will come with a dedicated PR there